hypothesis-python/tests/cover/test_health_checks.py

```diff
@@ -220,3 +227,49 @@ def test_lazy_slow_initialization_issue_2108_regression(data):
     # should be attributed to drawing from the strategy (not the test function).
     # Specificially, this used to fail with a DeadlineExceeded error.
     data.draw(LazyStrategy(slow_init_integers, (), {}))
+
+
+def test_does_not_trigger_health_check_on_simple_strategies(monkeypatch):
+    prng = Random(0)
+    existing_draw_bits = ConjectureData.draw_bits
+
+    # We need to make drawing data artificially slow in order to trigger this
+    # effect. This isn't actually slow because time is fake in our CI, but
+    # we need it to pretend to be.
+    def draw_bits(self, n, forced=None):
+        time.sleep(0.001)
+        return existing_draw_bits(self, n, forced)
+
+    monkeypatch.setattr(ConjectureData, "draw_bits", draw_bits)
+
+    for _ in range(100):
+        with deterministic_PRNG(prng.getrandbits(32)):
```

Comment:
Is there a particular reason to re-seed the PRNG on every iteration, instead of seeding it once outside the loop?

(The only reason I can think of is if you want each iteration's randomness to remain stable while debugging and tinkering?)

Reply:
Hmm, no not really. It seemed to make sense at the time but mostly because I was refactoring from writing them another way.